Steepfile

```diff
@@ -7,6 +7,8 @@ target :appsec do
   # check 'lib/datadog/kit'
 
   ignore 'lib/datadog/appsec/contrib'
```

Comment:
Note that this is ignored because of the dependency on actual Sinatra/Rails/Rack constants and methods, which are not typed. We can relax that `ignore` by minimally typing just what we need in `vendor/rbs`.